src/gt4py/next/program_processors/codegens/gtfn/codegen.py

```diff
@@ -260,6 +260,17 @@ def visit_Program(self, node: gtfn_ir.Program, **kwargs: Any) -> Union[str, Coll
     #include <gridtools/fn/${grid_type_str}.hpp>
     #include <gridtools/fn/sid_neighbor_table.hpp>
     #include <gridtools/stencil/global_parameter.hpp>
+    #include <gridtools/stencil/positional.hpp>
+    
+    // TODO(havogt): move to gtfn?
```

Comment:
I'd propose to wait until https://github.com/GridTools/gridtools/pull/1806 is merged next week.

Reply:
Once #1720 is in, we can remove this.

Reply:
Thanks, I removed that code.